scrivito_import_export/scrivito_import.rb

```diff
@@ -8,6 +8,22 @@ def import(dir_name:)
     api_key = ENV.fetch("SCRIVITO_API_KEY")
     api = RestApi.new(base_url, tenant, api_key)
 
+    custom_visibility_categories_file = File.join(dir_name, "custom_visibility_categories.json")
+    visibility_categories_ids_mapping = {}
+
+    if (File.exist?(custom_visibility_categories_file))
+      custom_visibility_categories = JSON.parse(File.read(custom_visibility_categories_file))
+      custom_visibility_categories.each do |visibility_category|
+        response = api.post("visibility_categories", {
+          "groups" => visibility_category["groups"], 
+          "title" => visibility_category["title"],
+          "description" => visibility_category["description"],
+          })
+        original_visibility_category_id = visibility_category["id"]
+        visibility_categories_ids_mapping[original_visibility_category_id] = response["id"]
+      end
+    end
```

Comment:
Should be moved to separate function. 
```suggestion
    visibility_categories_ids_mapping = get_visibility_categories_ids_mapping_from_file || {}
```

Reply:
I would preserve the "no mappings to read" state, i.e. don't default to `{}`. And even map an "imported" `{}` to `nil` using `.presence`, given there is no point in mapping ids using an empty hash either.